components/PostCard.tsx

```diff
@@ -58,15 +66,15 @@ export default function PostCard({ post, isLinkActive }: PostCardProps) {
             </Text>
           </Skeleton>
           <Text fontWeight="bold" fontSize="xl">
-            ${price}
+            ${price / 100}
```

Comment:
I think it would be nice to always show cents, even if they are 0 (see https://github.com/calebmech/Book-Bazar/pull/136/files#diff-65bded79d5b4cfd3a2245e76641f33ad1d90b634df743aca697802713ce16e6bR61). What do you think about this behaviour? (I am okay either way, I'll just need to adjust my code and the test case)